internal/cadence/linter.go

```diff
@@ -152,7 +170,7 @@ func (l *linter) lintFile(
 	report := func(diagnostic analysis.Diagnostic) {
 		diagnostics = append(diagnostics, diagnostic)
 	}
-	analysisProgram.Run(analyzers, report)
+	analysisProgram.Run(getAnalyzers(), report)
```

Comment:
Once the suggestion above is accepted:

```suggestion
	analysisProgram.Run(analyzers, report)
```